src/afrey/boot_asset_fingerprint.clj

```diff
@@ -67,6 +75,7 @@
                                        (:path)
                                        (get file-rename-hash)
                                        (io/file output-dir))]]
+                      :when (not (contains? (set (map :path sources)) (:path file)))]
```

Comment:
You've got an unbalanced square bracket here. You want to delete the last bracket on the line above, I believe